.github/workflows/pr_audit.yml

```diff
@@ -0,0 +1,18 @@
+name: PR Audit
+on: pull_request
+
+jobs:
+  textformat:
+    name: EditorConfig audit
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: CheckoutCode
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: EditorConfig-Action
+        uses: zbeekman/EditorConfig-Action@master
+        env:
+          ALWAYS_LINT_ALL_FILES: false
```

Comment:
I find maintaining these types of configurations confusing without some context to why settings are set a certain way.  Consider adding a comment noting if these settings are the defaults, or explain briefly why other settings if otherwise.